src/intelligence/components/DokanAI.tsx

```
@@ -349,259 +353,280 @@ const DokanAI = () => {
-                                        history={ getHistoryLabel(
+                                        element={
+                                            <div
+                                                className="mb-3 focus:outline-dokan-btn h-36 border rounded p-2.5 overflow-auto"
+                                                contentEditable={ true }
+                                                onBlur={ ( e ) => {
+                                                    inputHandler(
+                                                        e.target.innerHTML,
+                                                        'post_excerpt'
+                                                    );
+                                                } }
+                                                dangerouslySetInnerHTML={ {
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)